### PR TITLE
Fix 767

### LINKED ISSUE #767: processing doesn't update status
Currently the table is not being updated with status: "processing"

### PR DESCRIPTION
**Pull request by josenavas**
Fixes #767

### CODE REVIEW

**Comment by antgonza** [comment]
:+1: if test pass

File: qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html [hunk 1]
```diff
@@ -77,7 +77,12 @@ <h4 class="modal-title" id="myModalLabel">Choose processing parameters</h4>
       </tr>
   </table>
   {% if not processed_data or processing_status.startswith('failed') %}
-    <a class="btn btn-primary glyphicon glyphicon-play" data-toggle="modal" data-target="#process-modal-view-{{ppd_id}}" style="word-spacing: -10px;"> Process</a>
+    <a style="word-spacing: -10px;"
+    {% if processing_status == "processing" %}
+      class="btn btn-default glyphicon glyphicon-play" style=" pointer-events: none; cursor: default; background-color:grey;"
```
Comment (squirrelo) [inline qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html]: This doesn't seem to disable the button if it is processing. Need to disable it so we don't double-process

Comment (squirrelo) [inline qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html]: Oh woops it's pointing to a modal, so cool. Ignore this.

**Comment by squirrelo** [comment]
:+1:

**Pull request merged**